### PR TITLE
Added support of ODF comments from ';' at any position to the end of line https://github.com/GrandOrgue/grandorgue/issues/828

### PR DESCRIPTION
**Pull request by oleg68**
Resolves: #828

I've checked, that hauptwerk also can open files with comments starting at not the first position, so there are no reasons why to forbid it in GrandOrgue.

### CODE REVIEW

**Comment by larspalo** [comment]
The only drawback with this approach to comments is that you cannot have a ; in a text string anymore.

**Comment by oleg68** [comment]
> The only drawback with this approach to comments is that you cannot have a ; in a text string anymore.

May be to introduce an escaping or a quote-separated syntax of text fields (not to regard `;` in the quotas as a comment delimiter)?

**Comment by larspalo** [comment]
@oleg68 No, I don't think we should not complicate things too much. I think we can _either_:

1. Disallow any comments on lines that are read as strings, while just skipping them as comments on any other line
2. Document the behavior in help

_or_ just document in the help that semicolon cannot be used in strings in the odf (because they are parsed as comments)

**Comment by oleg68** [comment]
> @oleg68 > just document in the help that semicolon cannot be used in strings in the odf (because they are parsed as comments)

I like this choice. Moreover HW1 reads string keys only until the semicolon.

File: help/grandorgue.xml [hunk 1]
```diff
@@ -6259,7 +6259,8 @@ It is a text file in ISO-8859-1 encoding. The standard extension is
 UTF-8, if it starts with an appropriate byte order marker.
       </para>
       <para>
-Comment lines are started with <emphasis>;</emphasis> in the first column. Empty lines are ignored.
+        Comments are any text from <emphasis>;</emphasis> to the end of line.
```
Comment (larspalo) [inline help/grandorgue.xml:6262]: Thus it's not possible to include ; in any string.

**Review by larspalo — APPROVED** [review]

**Review by larspalo — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**